angular-holiday-planner/src/app/components/login/login.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="login-form w-screen h-screen flex justify-center items-center">
+  <form [formGroup]="loginForm" class="w-[20rem] pb-6 bg-slate-100 rounded-xl text-center">
+    <h1 class="text-center mt-5 font-extrabold text-2xl">Login</h1>
+
+    <div class="form-content flex flex-wrap mt-5">
+      <label class="w-full text-xl font-bold m-3 ">Email:</label>
+      <input formControlName="email" type="email" placeholder="Email" class="ml-12 w-56 h-7 pl-3">
+
+      <label class="w-full text-xl font-bold m-3">Password:</label>
+      <input formControlName="password" type="password" placeholder="Password" class="ml-12 w-56 h-7 pl-3">
```

Comment:
a11y issues my arise from not adding a name attribute to your inputs. Please test using the WAVE extenstion

Reply:
A11y will be accomodated for in a separate branch